core/adapters/http.go

```diff
@@ -63,6 +63,7 @@ type HTTPPost struct {
 	POST         models.WebURL   `json:"post"`
 	Headers      http.Header     `json:"headers"`
 	QueryParams  QueryParameters `json:"queryParams"`
+	Body         *JSONObj        `json:"body,omitempty"`
```

Comment:
I believe our JSON library already has a type for this, possibly `gjson.Result` ?

Reply:
Changed to string as recommended.  Well, a string reference, so it allows the user to override the payload to empty.  Hope this works.